src/Stryker.CLI/Stryker.CLI.UnitTest/ConfigFileTests.cs

```diff
@@ -70,19 +66,19 @@ public void StrykerCLI_WithConfigFile_ShouldStartStrykerWithConfigFileOptions(st
 
             mock.VerifyAll();
 
-            actualOptions.DevMode.ShouldBe(true);
-            actualOptions.AdditionalTimeoutMS.ShouldBe(9999);
-            actualOptions.LogOptions.LogLevel.ShouldBe(LogEventLevel.Verbose);
-            actualOptions.ProjectUnderTestNameFilter.ShouldBe("ExampleProject.csproj");
-            actualOptions.Reporters.ShouldHaveSingleItem();
-            actualOptions.Reporters.ShouldContain(Reporter.ConsoleReport);
-            actualOptions.ConcurrentTestrunners.ShouldBe(1);
-            actualOptions.Thresholds.Break.ShouldBe(20);
-            actualOptions.Thresholds.Low.ShouldBe(30);
-            actualOptions.Thresholds.High.ShouldBe(40);
-            actualOptions.FilePatterns.Count().ShouldBe(2);
-            actualOptions.FilePatterns.ShouldContain(filePattern);
-            actualOptions.Optimizations.ShouldBe(OptimizationFlags.CoverageBasedTest | OptimizationFlags.AbortTestOnKill);
+            actualInputs.AdditionalTimeoutMsInput.SuppliedInput.ShouldBe(9999);
+            actualInputs.VerbosityInput.SuppliedInput.ShouldBe("trace");
```

Comment:
Do you really need the `Input` suffix on every prop, given that the class is already called `StrykerInputs`?